sdk/python/docs/source/feast.rst

```diff
@@ -169,14 +169,6 @@ feast.field module
    :undoc-members:
    :show-inheritance:
 
-feast.flags module
```

Comment:
does `sdk/python/docs/index.rst` also need to be updated? not sure which of these two is the source of truth

Reply:
nope. They are just different. index.rst gives the top level structure and this isn't referenced there.

This change was generated by running `make build-sphinx`